src/modules/rabbitmq/elastic-updates/elastic-updates-events.consumer.ts

```diff
@@ -18,7 +18,7 @@ export class ElasiticUpdatesConsumer {
       const mintNftEvents = events?.events?.filter((e: { identifier: NftEventEnum }) => e.identifier === NftEventEnum.ESDTNFTCreate);
       const burnAndUpdateNftAttributesEvents = events?.events?.filter(
         (e: { identifier: NftEventEnum }) =>
-          e.identifier === NftEventEnum.ESDTNFTBurn || e.identifier === NftEventEnum.ESDTNFTUpdateAttributes,
+          e.identifier === NftEventEnum.ESDTNFTBurn || e.identifier === NftEventEnum.ESDTNFTUpdateAttributes || e.identifier === NftEventEnum.ESDTMetaDataUpdate || e.identifier === NftEventEnum.ESDTMetaDataRecreate,
```

Comment:
Can have a collection of identifiers of interest, then do: `arr.includes(value)`. Though, if this approach is used in other places, keep it for consistency.